static/app/components/feedback/feedbackItem/screenshotSection.tsx

```diff
@@ -0,0 +1,114 @@
+import {useState} from 'react';
+
+import {
+  useDeleteEventAttachmentOptimistic,
+  useFetchEventAttachments,
+} from 'sentry/actionCreators/events';
+import {openModal} from 'sentry/actionCreators/modal';
+import Section from 'sentry/components/feedback/feedbackItem/feedbackItemSection';
+import {t} from 'sentry/locale';
+import type {Organization} from 'sentry/types';
+import type {Event} from 'sentry/types/event';
+import type {EventAttachment} from 'sentry/types/group';
+import {objectIsEmpty} from 'sentry/utils';
+
+import Screenshot from '../../events/eventTagsAndScreenshot/screenshot';
+import Modal, {modalCss} from '../../events/eventTagsAndScreenshot/screenshot/modal';
+
+const SCREENSHOT_NAMES = [
+  'screenshot.jpg',
+  'screenshot.png',
+  'screenshot-1.jpg',
+  'screenshot-1.png',
+  'screenshot-2.jpg',
+  'screenshot-2.png',
+];
```

Comment:
What is this used for?

Reply:
oh i accidentally copied this over from issues, let me remove it